### DISCUSSION TITLE
Installing packaged 3.1.1 on Mac OS X fails

### DISCUSSION THREAD

**Discussion by fgeorges** [apache/couchdb#3326] [Q&A]
Hi,

I went to https://couchdb.apache.org/#download and downloaded the version for "macOS (10.10+)."  Once unzipped, when I double-click `Apache CouchDB.app`, a new item appears on the menu bar of the OS, but neither `About Apache CouchDB` nor `View Logs` do anything.

It tries to open Fauxton at http://localhost:5984/_utils, every 5 or 10 seconds, but there is nothing running on that port (error ERR_CONNECTION_REFUSED on the browser).

Since the `View Logs` menu item does not work, it is quite hard to investigate.  Any idea where I can find them?  How can I find any more info?

Thanks!

<img width="288" alt="Screen Shot 2021-01-13 at 19 57 46" src="https://user-images.githubusercontent.com/584411/104497925-578d2a00-55db-11eb-863f-6dd3699e7993.png">

**Comment by janl** [comment]
can you share more information about your mac, like the type of machine, specs, software version, etc.

also, can you share the output of `ps ax | grep beam` in the terminal?
